bitwarden_license/src/Commercial.Core/SecretManagerFeatures/AccessTokens/CreateAccessTokenCommand.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Repositories;
+using Bit.Core.SecretManagerFeatures.AccessTokens.Interfaces;
+using Bit.Core.Utilities;
+
+namespace Bit.Commercial.Core.SecretManagerFeatures.AccessTokens;
+
+public class CreateAccessTokenCommand : ICreateAccessTokenCommand
+{
+    private readonly IApiKeyRepository _apiKeyRepository;
+
+    public CreateAccessTokenCommand(IApiKeyRepository apiKeyRepository)
+    {
+        _apiKeyRepository = apiKeyRepository;
+    }
+
+    public async Task<ApiKey> CreateAsync(ApiKey apiKey)
+    {
+        apiKey.ClientSecret = CoreHelpers.SecureRandomString(30);
```

Comment:
Can we make `30` a constant with a clear name so we don't have magic numbers? It's probably clear what the `30` is for, but not *why* `30` was picked (can you have any length, is there a min required length, etc). We know from other areas but a new reader might not know.

This isn't a big deal though - just a small idea.

Reply:
resolved in https://github.com/bitwarden/server/pull/2411/commits/69276b76b0e4931572168f39665f63ce98c90705